websites/L/Learnjavascript/presence.ts

```diff
@@ -0,0 +1,257 @@
+const presence = new Presence({
+	clientId: "1024006470364315668",
+}),
+browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+const presenceData: PresenceData = {
+	largeImageKey: "javascript",
```

Comment:
Use an Imgur link for `largeImageKey` for easier maintainability in the future.

Reply:
So basically I should paste https://i.imgur.com/03uHFM5.png into the value of largeImageKey instead of "javascript"?